docs/usage.rst

```diff
@@ -3,17 +3,19 @@ Usage
 
 OpenSCM defines and provides three interfaces for its users.
 
-Its **Core** interface is targeted to users who want to include
+Its **core** interface is targeted to users who want to include
 OpenSCM in their model, e.g. integrated assessment modellers. This
 interface provides the basic functionality necessary to run all SCMs
 included in OpenSCM. It includes functions for getting and setting
-parameters as well as to run and reset the model.
+parameters as well as to run and reset the model. Additionally, it
+allows for reading and writing parameters from and to standardized and
+other file formats, including whole scenario definitions.
 
-The **Pythonic** interface adds several convenience functions on top
-of the core interface. It allows for reading and writing parameters
-from and to standardized and other file formats, including whole
-scenario definitions. It also provides functions for running ensembles
-of model runs.
+The **ensemble** interface provides functions for running ensembles of
+model runs.
+
+..
+    TODO: describe ScmDataFrame
```

Comment:
@swillner this would be my suggested text:

> The **ScmDataFrame** provides a high-level model output analysis tool. It is designed to make model output filtering, plotting and writing to/from disk as easy as possible. In addition, it integrates with the tools provided by the `pyam <https://github.com/IAMconsortium/pyam>`_ package which facilitates easy inclusion of data from integrated assessment modelling exercises too.

Reply:
added ;)